### PR TITLE
Changed dockerfile to run on Deis

### PR DESCRIPTION
**Pull request by illegalprime**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,10 @@
-FROM node:alpine
+FROM andthensome/alpine-node-bash
```
Comment (petschekr) [inline Dockerfile]: Just wondering, why did you change the base image from the official image?

Comment (illegalprime) [inline Dockerfile]: Deis requires you to have an image that:
1. uses the `EXPOSE` directive to expose only one port
2. that port must be listening for an HTTP connection
3. the image must use the `CMD` directive to define the entry point
4. the image **must** contain `bash`

Comment (petschekr) [inline Dockerfile]: Ah makes sense. I looked at the docs for Alpine linux and it seems you can install bash by putting `RUN apk add bash bash-doc bash-completion` in the dockerfile if we want to stick to the official images. It's up to you though.

Comment (illegalprime) [inline Dockerfile]: great point, I like the official ones better too.
